compiler/src/dotty/tools/dotc/ast/NavigateAST.scala

```diff
@@ -72,7 +72,7 @@ object NavigateAST {
    *                         end point are the same, so this is useful when trying to reconcile
    *                         nodes with source code.
    */
-  def pathTo(span: Span, from: Positioned, skipZeroExtent: Boolean = false)(using Context): List[Positioned] = {
```

Comment:
The documentation comment above needs to be slightly tweaked: it says "from node `from`" but I guess now it would be "from any node in `from`"

Reply:
Changed. Good catch!